ui/cockpit/src/modules/components/CancelProcessInstance/CancelProcessInstance.js

```diff
@@ -0,0 +1,324 @@
+            exclusive: ["type"]
+          });
+        });
+      });
+  };
+
+  return (
+    <>
+      <ActionButton
+        labels={{
+          enabled: "PLGN_HIST_CANCEL_RUNNING_INSTANCES",
+          disabled: "PLGN_HIST_CANCEL_NOT_POSSIBLE"
+        }}
+        icon="remove"
+        onClick={openDialog}
+        disabled={check(STATUS.FAILED) || check(STATUS.LOADING)}
+      />
+
+      <Modal show={show} onHide={closeDialog}>
+        <Modal.Header>
+          <h3>{translate("PLUGIN_CANCEL_PROCESS_DELETE_PROCESS_INSTANCE")}</h3>
+        </Modal.Header>
+        <Modal.Body>
+          <Notifications />
+
+          {check(STATUS.LOADING) && (
+            <>
+              <span className="glyphicon glyphicon-loading" />
+              {translate("PLUGIN_CANCEL_PROCESS_LOADING_INFORMATION")}
+            </>
+          )}
+
+          {(check(STATUS.INITIAL) || check(STATUS.ACTION_PENDING)) && (
+            <>
+              {subProcessInstances.length > 0 && (
+                <>
+                  {translate("PLUGIN_CANCEL_PROCESS_BEFORE_DELETION")}
+                  <Table className="table cam-table">
+                    <thead>
+                      <tr>
+                        <th className="instance-id uuid">
+                          {translate("PLUGIN_CANCEL_PROCESS_ID")}
+                        </th>
+                      </tr>
+                    </thead>
+
+                    <tbody>
+                      {subProcessInstances.map(subProcessInstance => (
+                        <tr key={subProcessInstance.id}>
+                          <td className="instance-id uuid">
+                            <a
+                              href={`#/process-instance/${
+                                subProcessInstance.id
+                              }`}
+                              target="_blank"
+                              rel="noopener noreferrer"
+                            >
+                              {subProcessInstance.id}
+                            </a>
+                          </td>
+                        </tr>
+                      ))}
+                      {subProcessInstancesCount >
+                        subProcessInstances.length && (
+                        <tr>
+                          <td>
+                            {translate(
+                              "PLUGIN_CANCEL_PROCESS_OTHER_PROCESS_INSTANCES",
+                              {
+                                count:
+                                  subProcessInstancesCount -
+                                  subProcessInstances.length
+                              }
+                            )}
+                          </td>
+                        </tr>
+                      )}
+                    </tbody>
+                  </Table>
+                </>
+              )}
+              {(!getConfig().skipIoMappings.hidden ||
+                !getConfig().skipCustomListeners.hidden) && (
```

Comment:
We can add accessor functions to the config file for the skipCustomListener config if it makes things easier

Reply:
👍